internal/command/action.go

```diff
@@ -67,6 +71,14 @@ func runAction(ac *ActionConfig) error {
 		err     error
 	)
 
+	if ac.AllProvVersions{
+		err := printProviderVersions()
+		if err != nil {
+			return fmt.Errorf(err.Error())
```

Comment:
You can just return the err here I think without the need to format it. In fact, I think you literally can just do this:
```
if ac.AllProvVersions{
    return printProviderVersions()
}
```

Reply:
Good catch

Reply:
Had to keep err in there for testing